### PR TITLE
feat: checks if msfs is open

### LINKED ISSUE #13: [REQUEST] Should not install when MSFS 2020 is already running
**Is your feature request related to a problem? Please describe.**

**Describe the feature you'd like to see implemented**

Add a logic to not allow the user to install the mod when MSFS is already running.

**Additional context**

Discord username (if different from GitHub):

### PR DESCRIPTION
**Pull request by ZigTag**
Fixes #13 

## Summary of Changes

I added a install button which says "Please Close MSFS" 
also made the install bar smoother.

## Additional context

Discord username (if different from GitHub):

### CODE REVIEW

File: src/renderer/components/AircraftSection/index.tsx [hunk 1]
```diff
@@ -49,14 +50,16 @@ const index: React.FC<Props> = (props: Props) => {
...
 
     const isDownloading = download?.progress >= 0;
 
     useEffect(() => {
         checkForUpdates(selectedTrack);
-
+        checkIfMSFS();
     },
 
     []);
```
Comment (Benjozork) [inline src/renderer/components/AircraftSection/index.tsx:65]:
```suggestion
    }, []);
```

File: src/renderer/components/AircraftSection/index.tsx [hunk 2]
```diff
@@ -89,6 +92,16 @@ const index: React.FC<Props> = (props: Props) => {
         }
     }
 
+    async function checkIfMSFS() {
```
Comment (Benjozork) [inline src/renderer/components/AircraftSection/index.tsx:95]: Why is this `async` ?

File: src/renderer/components/AircraftSection/styles.tsx [hunk 3]
```diff
@@ -191,3 +191,16 @@ export const InstalledButton = styled(
...
+        <InstallButton
+            icon={null}
+            style={{
+                color: "#dddddd",
+                background: "#bc0d0d",
+                borderColor: "#bc0d0d",
+                pointerEvents: "none"
+            }}
+            {...props}
+        >Please Close MSFS</InstallButton>)``;
```
Comment (Benjozork) [inline src/renderer/components/AircraftSection/styles.tsx:206]:
```suggestion
        >Please close MSFS</InstallButton>)``;
```

**Review by Benjozork — CHANGES_REQUESTED** [review]

File: src/renderer/components/AircraftSection/index.tsx (hunk 2 above)
Comment (ZigTag) [reply src/renderer/components/AircraftSection/index.tsx:95]: So it won't block the other functions

Comment (Benjozork) [reply src/renderer/components/AircraftSection/index.tsx:101]: Discussed on discord

File: src/renderer/components/AircraftSection/index.tsx [hunk 4]
```diff
@@ -94,6 +98,16 @@ const index: React.FC<Props> = (props: Props) => {
...
```
Comment (nistei) [inline src/renderer/components/AircraftSection/index.tsx:101]:
```suggestion
    function checkIfMSFS() {
```

**Review by nistei — CHANGES_REQUESTED** [review]

**Pull request merged**